src/cba-list-new/cba-list-new.js

```diff
@@ -22,9 +22,10 @@ class List extends HTMLElement
     this.attachShadow({mode: "open"});
     this.shadowRoot.innerHTML = `
     <div id="container">
-      <h2></h2>
-      <h3 id="column"><a href="#"></a></h3>
+      <h1></h1>
+      <p id="column"><a href="#"></a></p>
```

Comment:
Why we need this `p` ?

Reply:
If we are not using #column anymore we can remove this one also. OK?